batch_jobs/utils/utils.py

```diff
@@ -0,0 +1,80 @@
+import csv
+import logging
+import boto3
+from botocore.exceptions import ClientError
+
+
+def write_csv(filename, files, fieldnames=None):
```

Comment:
nitpick: if we are hard-coding as tab separated, we may want to name function write_tsv